### PR TITLE
Local execution

### PR DESCRIPTION
**Pull request by Eitu33**

### CODE REVIEW

File: massa-execution-worker/src/tests/scenarios_mandatories.rs [hunk 1]
```diff
@@ -321,6 +321,98 @@ fn send_and_receive_async_message() {
     manager.stop();
 }
 
+#[test]
```
Comment (AurelienFT) [inline massa-execution-worker/src/tests/scenarios_mandatories.rs:324]: I think it's cool if we always have the walkthrough like in the others tests !

**Review by AurelienFT — COMMENTED** [review]
Nice one.

File: massa-execution-worker/src/tests/scenarios_mandatories.rs (hunk 1 above)
Comment (Eitu33) [reply massa-execution-worker/src/tests/scenarios_mandatories.rs:335]: Done

**Review by AurelienFT — APPROVED** [review]
"Mergeable" when the dependencies are merged

**Pull request merged**